src/modules/dashboard/components/ColonyTokenManagementDialog/ColonyTokenManagementDialog.tsx

```diff
@@ -8,18 +8,21 @@ import {
   ColonyTokensQueryVariables,
 } from '~data/index';
 import { Address } from '~types/index';
-
-import { tokenIsETH } from '../../../core/checks';
+import tokensList from './tokenlist.json';
 
 interface Props {
   colonyAddress: Address;
   cancel: () => void;
   close: () => void;
 }
 
-const displayName = 'dashboard.ColonyTokenEditDialog';
+const displayName = 'dashboard.ColonyTokenManagementDialog';
 
-const ColonyTokenEditDialog = ({ colonyAddress, cancel, close }: Props) => {
+const ColonyTokenManagementDialog = ({
+  colonyAddress,
```

Comment:
This component will receive the full colony object when it will be wired with index modals and the rest of the wizard.

Please refactor this to reflect that. (so this doesn't need to fetch the colony data internally anymore)